config/locales/sv.yml

```diff
@@ -39,6 +39,7 @@ sv:
   search: &search Sök
   close: Stänga
   lastupdated: &last_updated Senast uppdaterad
+  action: Verkan
```

Comment:
What's the context? if only admin then it's fine. otherwise its a different word depending on context

Reply:
This is a column header for actions the user can take:

1. Associate themselves with one or more subcategories

2. Bring up an email form to send a request to the admin to add subcategory(s) to a category.

Reply:
Does it need a header? 
-- No :)